### PR TITLE
add accessibility configuration

### LINKED ISSUE #453: Add accessibilityIdentifier to all items
To make XCUITests more resiliant to copy changes, and making L10n screenshots easier to write, each UIView that the user will interact with should have an `accessibilityIdentifier`.

To make a more accessible UI, each UIView that the user will interact with should have an `accessibilityLabel`.

### LINKED ISSUE #454: Add VoiceOver to custom alerts
From #441

### PR DESCRIPTION
**Pull request by sashei**
resolves #453 
resolves #454

### CODE REVIEW

**Comment by isabelrios** [comment]
This looks great, there are only a few things fyi I would mention just in case you have not seen:
- I was not able to tap on the Cancel option when typing something on the Search text field, it says like you are there although you tap on the Cancel button
- I set the Auto Lock to 1 minute and the app locked while I was interacting with it (the voice was saying something about the option selected and the app locked, the voice was still on the background)

Let me know if you want me to check it again or pay attention to a particular feature/thing. Thanks!

File: lockbox-iosTests/ItemDetailViewSpec.swift [hunk 1]
```diff
@@ -83,6 +83,7 @@ class ItemDetailViewSpec: QuickSpec {
                         ItemDetailCellConfiguration(
                                 title: Constant.string.webAddress,
                                 value: "www.meow.com",
+                                accessibilityLabel: "something accessible",
```
Comment (jhugman) [inline lockbox-iosTests/ItemDetailViewSpec.swift]: Suggest making these vary somehow.

File: lockbox-ios/Common/Extensions/UIViewController+.swift [hunk 2]
```diff
@@ -71,9 +72,11 @@ extension UIViewController: SpinnerAlertView {
 
             spinnerAlertView.activityIndicatorView.startAnimating()
             self.animateAlertIn(spinnerAlertView)
+            UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, Constant.string.syncingYourEntries)
 
             dismiss
                     .drive(onNext: { _ in
+                        UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, Constant.string.doneSyncingYourEntries)
```
Comment (jhugman) [inline lockbox-ios/Common/Extensions/UIViewController+.swift]: 😍

**Comment by devinreams** [comment]
>I was not able to tap on the Cancel option when typing something on the Search text field, it says like you are there although you tap on the Cancel button

Good catch.

@sashei do you think that’s something that can easily be addressed in this PR?

>Firefox Logo on Welcome screen

Good catch, I thought I _had_ confirmed this wasread when I tested it but maybe I’m crazy...

@sashei is that also something that can be added easily here?

>app locked while using it

Uh oh.. I don’t believe this is related because I experienced the same recently on another branch... but can’t find the steps to reproduce it and didn’t get locked out while testing again here after you all. 

The fact you both did run into this is troubling but feels like “yet another” timer edge case and unrelated. 

What do you think @sashei?

>”Learn More” link in Usage Data

Ah, the way to access this is in the “hidden” switcher.

If you tap on the row in the table (it reads the whole text) then swipe up on just that table row it will read “Learn More” which means you “switched” to selecting the learn more link within the cell. Then you can immediately double tap to activate/click the link.

>Open Websites in ... dimmed

Ah yes, it would be nice to add some text that says something like “Firefox Focus, disabled, install the app to make it available” or similar.

@sashei is that something that can easily be added?

**Comment by sashei** [comment]
@devinreams hi, yep, addressing PR comments this morning.

**Comment by sashei** [comment]
@devinreams @isabelrios @nickbrandt it looks like the cancel button bit is tricky because of the selective hide / show behavior. I'm going to just file a follow-up issue to make that particular element accessible.

**Pull request merged**